opencti-platform/opencti-graphql/src/domain/user.js

```diff
@@ -623,7 +644,11 @@ export const addUser = async (context, user, newUser) => {
   await Promise.all(relationGroups.map((relation) => createRelation(context, user, relation)));
   // Audit log
   if (isCreation) {
-    const actionEmail = ENABLED_DEMO_MODE ? REDACTED_USER.user_email : newUser.user_email;
+    let actionEmail;
+    actionEmail = ENABLED_DEMO_MODE ? REDACTED_USER.user_email : newUser.user_email;
+    if (userServiceAccount) {
+      actionEmail = actionEmail || `automatic+${uuid()}@opencti.invalid`;
+    }
```

Comment:
```suggestion
    actionEmail = ENABLED_DEMO_MODE ? REDACTED_USER.user_email : element.user_email;
```
I think it's better to use the email that has actually been created in database. To check but should be in element.